packages/compass/src/app/styles/index.less

```diff
@@ -33,6 +38,9 @@ body {
   height: 100vh;
```

Comment:
We are going to move these to css-in-js with some `Global` stuff in a different PR?